### PR TITLE
Initial proposal for react tag conventions

### PR DESCRIPTION
**Pull request by lededje**

### CODE REVIEW

File: _guidelines/react.md [hunk 1]
```diff
@@ -64,3 +64,30 @@ Avoid using an array index as `key` prop, prefer a unique ID: using the index as
...
+
+```
+// bad
+<Foo className="stuff"></Foo>
+
+// good
+<Foo className="stuff" />
+```
+
+If your component has multi-line properties, close its tag on a new line. eslint: [`react/jsx-closing-bracket-location`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-closing-bracket-location.md)
```
Comment (kapowaz) [inline _guidelines/react.md]: Footnotes?

**Comment by martyphee** [comment]
LGTM

File: _guidelines/react.md [hunk 2]
```diff
@@ -64,3 +64,30 @@ Avoid using an array index as `key` prop, prefer a unique ID: using the index as
   />
 ))}
 ```
+
+## Tags
+
+Always self-close tags that have no children. eslint: [`react/self-closing-comp`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md)
```
Comment (edds) [inline _guidelines/react.md]: Why? Can we add some reasoning for why this is better?

Comment (lededje) [inline _guidelines/react.md]: This one is just because it looks better.
`<View></View>` vs `<View />`

Fairly standard in html

**Pull request merged**